meshio/_abaqus.py

```diff
@@ -90,7 +91,7 @@
 def read(filename):
     """Reads a Abaqus inp file.
     """
-    with open(filename, "r") as f:
+    with open_file(filename, "r") as f:
         out = read_buffer(f)
```

Comment:
Do we still need the separation between `read` and `read_buffer` now? (Also flac3d etc.)

Reply:
Some code somewhere might be relying on it existing, for cases where people were already reading out of HTTP responses and so on. It might also be nice to have an "accepts anything" function and then the actual reading logic in a more type-safe function, as it is now.

Reply:
Alright.